app/Nova/ActionGuards/UitIdv1/ActivateUitIdv1ClientGuard.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\ActionGuards\UitIdv1;
+
+use App\Nova\ActionGuards\ActionGuard;
+use App\UiTiDv1\UiTiDv1ClusterSDK;
+use App\UiTiDv1\UiTiDv1Consumer;
+use App\UiTiDv1\UiTiDv1ConsumerStatus;
+
+final class ActivateUitIdv1ClientGuard implements ActionGuard
+{
+    public function __construct(
+        private readonly UiTiDv1ClusterSDK $sdk,
+    ) {
+    }
+
+    public function canDo(object $resource): bool
+    {
+        if (!$resource instanceof UiTiDv1Consumer) {
+            return false;
+        }
+
+        return $this->sdk->fetchStatusOfConsumer($resource) === UiTiDv1ConsumerStatus::Blocked;
```

Comment:
Not sure if an activate should be allowed on more status values, maybe a better check is `!==  UiTiDv1ConsumerStatus::Active`

Reply:
Would rather do not do this. The current other possible value is "unknown", which probably means something went wrong with the api-call, for example the key does not actually exist. Showing an activate button then would be confusing, because it could never work.
I can check for not active and not unknown as alternative?

Reply:
I checked the docs and `ACTIVE` and `BLOCKED` are the only possible states, so keep this check indeed.

That also makes me wonder why we need `UNKNOWN`

Reply:
If there was something wrong with the API call we should get an exception.